assets/js/components/Dashboard/PaMessageForm/SelectStationsAndZones/SelectZonesPage.tsx

```diff
@@ -66,10 +68,18 @@ const SelectZonesPage = ({
         return routeID;
       }),
     )(places);
-  }, [places, routeToRouteIDMap, value]);
+  };
+
+  // eslint-disable-next-line react/hook-use-state
+  const [initialPlacesWithSelectedSigns] = useState<Place[]>(
+    getInitialPlacesWithSelectedSigns,
+  );
+
+  // eslint-disable-next-line react/hook-use-state
+  const [initialRoutes] = useState(getInitialRoutes);
```

Comment:
Was able to rework that function so it looked at the new unchanging state value instead of the ever-changing `value` prop. Keeps the filter list from having the same problem as the place rows.